### ISSUE TITLE
gnt-cluster verify returning wrong exit code

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#800]
Originally reported of Google Code with ID 744.

```
In the situation described by Issue 742, "gnt-cluster verify" returns 0 as the exit code notwithstanding the error during its execution.
```
Originally added on 2014-03-03 17:33:23 +0000 UTC.

Mentioned Issue with GoogleCode ID 742 has been migrated to GitHub with ID #798.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-03-03 17:35:29 +0000 UTC.

Added Labels: Priority-High
Added to Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 510f672fafdfb00ec3ad3d15d49ce61639efe4ce
Author: Michele Tartara <mtartara@google.com>
Date:   Mon Mar 3 18:36:36 2014 +0100

    Fix error propagation in post-commit hooks
    
    An error in the post-commit hooks could not be propagated correctly and could
    result in e.g. the return code of gnt-cluster verify to be 0 even in presence of
    an error in its output.
    
    Fixes Issue 744.
    
    Signed-off-by: Michele Tartara <mtartara@google.com>
    Reviewed-by: Thomas Thrainer <thomasth@google.com>

```
Originally added on 2014-03-04 09:45:48 +0000 UTC.

Mentioned Issue with GoogleCode ID 744 has been migrated to GitHub with ID #800.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released in 2.10.1
```
Originally added on 2014-03-05 13:55:53 +0000 UTC.

Changed State: Released
